src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    private static final int MIN_CAPACITY = 8000;
+    @NotNull
+    @Min(value = MIN_CAPACITY)
     private int capacity;
```

Comment:
no point for putting `@NotNull` over primitives

Reply:
done